scripts/cache-olean.py

```diff
@@ -75,6 +75,15 @@ def fetch_mathlib(asset):
         ar.extractall('.')
         print("... successfully extracted olean archive.")
 
+def should_proceed_with_dirty_repo(action):
+    if action == ['--fetch-even-if-dirty']:
+        return True
+    if action == ['--fetch']:
+        print('Your repo is dirty; fetching in this state could cause you to lose data. '\
+              'If you understand the risks, you can use --fetch-even-if-dirty.')
+    else:
+        print('Your repo is dirty; you must commit or discard your changes to perform this action.')
```

Comment:
Use `'Your repo is dirty; commit or discard your changes to perform this action.'` instead.